modules/verible/0.0.3890/MODULE.bazel

```diff
@@ -0,0 +1,78 @@
+module(
+    name = "verible",
+    version = "0.0.3890",
+    compatibility_level = 0
+)
+
+bazel_dep(name = "platforms", version = "0.0.10")
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "rules_license", version = "1.0.0")
+bazel_dep(name = "nlohmann_json", version = "3.11.3.bcr.1")
+bazel_dep(name = "re2", version = "2024-07-02.bcr.1")
+bazel_dep(name = "zlib", version = "1.3.1.bcr.3")
+
+# Register m4/flex/bison rules and toolchains
+bazel_dep(name = "rules_m4", version = "0.2.3")
+
+m4 = use_extension(
+    "@rules_m4//m4/extensions:m4_repository_ext.bzl",
+    "m4_repository_ext",
+)
+m4.repository(
+    name = "m4",
+    extra_copts = ["-O3"],
+    version = "1.4.18",
+)
+use_repo(m4, "m4")
+
+register_toolchains("@m4//:toolchain")
+
+bazel_dep(name = "rules_flex", version = "0.3")
+
+flex = use_extension(
+    "@rules_flex//flex/extensions:flex_repository_ext.bzl",
+    "flex_repository_ext",
+)
+flex.repository(
+    name = "flex",
+    extra_copts = ["-O3"],
+    version = "2.6.4",
+)
+use_repo(flex, "flex")
+
+register_toolchains("@flex//:toolchain")
+
+bazel_dep(name = "rules_bison", version = "0.3")
+
+bison = use_extension(
+    "@rules_bison//bison/extensions:bison_repository_ext.bzl",
+    "bison_repository_ext",
+)
+bison.repository(
+    name = "bison",
```

Comment:
Is this usage safe if someone else uses the same name? Can you use a name unique to your module instead?

That would still cause issues if someone brings in multiple versions of your module, so ideally rules_bison would offer guidance on how to avoid collisions.

Reply:
Turns out all the additional `bison.repository()` stuff was not needed, the `bazel_dep()`s are sufficient.
Fixed upstream.